packages/gatsby/src/utils/worker/__tests__/share-state.ts

```diff
@@ -15,8 +17,10 @@ const dummyPagePayload = {
 }
 
 describe(`worker (share-state)`, () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     store.dispatch({ type: `DELETE_CACHE` })
+    const fileDir = path.join(process.cwd(), `.cache/redux`)
+    await fs.emptyDir(fileDir)
```

Comment:
do we still need this in tests that were not touched in this PR? I'd hope we don't?